src/lib/features/Footer.stories.svelte

```diff
@@ -3,19 +3,11 @@
 
   import Footer from './Footer.svelte';
 
-  const { Story } = defineMeta({ title: 'UI Components/Features/Footer', component: Footer });
+  const { Story } = defineMeta({ title: 'UI Components/Features', component: Footer });
 </script>
 
-<!--Primary-->
-<Story name="Footer - Light Background (Gray example)" id="footerLight">
-  <div class="bg-gray-100 p-6">
-    <Footer type="light" />
-  </div>
-</Story>
-
-<!--Primary-->
-<Story name="Footer - Dark Background (Navy example)" id="footerDark">
-  <div class="bg-navy p-6">
-    <Footer type="dark" />
-  </div>
+<Story name="Default Footer" args={Footer.props}>
+  {#snippet children(args)}
+    <Footer {...args} />
```

Comment:
I thought we had a dark and light footer, but the dark/light control doesn't change anything